apps/marginfi-v2-ui/src/components/Points/PointsLeaderBoard/PointsBoardHeader/PointsBoardHeader.consts.ts

```diff
@@ -0,0 +1,51 @@
+interface HeadCell {
+  disablePadding: boolean;
+  id: string;
+  label: string;
+  numeric: boolean;
+}
+
+export const headCells: readonly HeadCell[] = [
+  {
+    id: "rank",
+    numeric: false,
+    disablePadding: false,
+    label: "Rank",
+  },
+  {
+    id: "id",
+    numeric: false,
+    disablePadding: false,
+    label: "User",
+  },
+  {
+    id: "total_activity_deposit_points",
+    numeric: true,
+    disablePadding: false,
+    label: "Lending Points",
+  },
+  {
+    id: "total_activity_borrow_points",
+    numeric: true,
+    disablePadding: false,
+    label: "Borrowing Points",
+  },
+  {
+    id: "total_referral_borrow_points",
```

Comment:
nit - the `id` here should just generally refer to the head cell (not to what shows in the actual data), but technically this column will be deposit + borrow referral points